<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoExpandedView</class>
 <widget class="QWidget" name="VideoExpandedView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">VideoExpandedView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>10</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="VideoLarge" name="expanded_video">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame_rate" stdset="0">
       <number>10</number>
      </property>
     </widget>
     <widget class="QWidget" name="bottom_layout" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="AlertLog" name="alert_log"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="config_button_layout">
         <item>
          <widget class="QPushButton" name="stream_config_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Toggle Stream Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task_config_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Task Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="capsule_stream_config_button">
           <property name="text">
            <string>Stream Capsule Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_stream_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete Stream</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoLarge</class>
   <extends>QGraphicsView</extends>
   <header>brainframe_qt.ui.main_window.video_expanded_view.video_large.video_large</header>
  </customwidget>
  <customwidget>
   <class>AlertLog</class>
   <extends>QWidget</extends>
   <header>brainframe_qt.ui.main_window.video_expanded_view.alert_log.alert_log</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>task_config_button</sender>
   <signal>clicked()</signal>
   <receiver>VideoExpandedView</receiver>
   <slot>open_task_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1051</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_stream_button</sender>
   <signal>clicked()</signal>
   <receiver>VideoExpandedView</receiver>
   <slot>delete_stream_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1051</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>719</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>capsule_stream_config_button</sender>
   <signal>clicked()</signal>
   <receiver>VideoExpandedView</receiver>
   <slot>open_stream_capsule_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>954</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>expanded_stream_closed_slot()</slot>
  <slot>open_task_config()</slot>
  <slot>open_source_config()</slot>
  <slot>delete_stream_button_clicked()</slot>
  <slot>open_stream_capsule_config()</slot>
 </slots>
</ui>
